upstream-configuration/patches.webspark.json

```diff
@@ -1,7 +1,8 @@
 {
     "drupal/core": {
         "#2951547: Fix issue with layout overflow": "https://www.drupal.org/files/issues/2022-07-13/f245f21ea664f22c81d8d28c6f0f4a42fb9a5890.patch",
-        "3109650: Refactor Xss::attributes() to allow filtering of style attribute values": "https://www.drupal.org/files/issues/2021-09-22/core-9.2-xss-refactor_filter_attributes-3109650-43.patch"
+        "3109650: Refactor Xss::attributes() to allow filtering of style attribute values": "https://www.drupal.org/files/issues/2021-09-22/core-9.2-xss-refactor_filter_attributes-3109650-43.patch",
+        "3405976: Transaction autocommit during shutdown": "https://git.drupalcode.org/project/drupal/-/commit/b09cf3ba80f5ae65e101cb77e438c0a7e8d36a34.patch"
```

Comment:
This patch has now been merged into Drupal 10.2.4 - instead of adding the patch, let's upgrade the Drupal version.